torchbenchmark/util/backends/blade.py

```diff
@@ -63,4 +62,10 @@ def blade(model: 'torchbenchmark.util.model.BenchmarkModel', backend_args: List[
             allow_tracing=True,
             model_inputs=tuple(example_inputs),
         )
+    if isinstance(optimized_model, torch.jit.ScriptModule):
+        for attr_name in getattr(model, "blade_reserve_attrs", []):
+            attr = getattr(module, attr_name, None)
+            if isinstance(attr, int):
+                optimized_model._c._register_attribute(attr_name, torch._C.IntType.get(), attr)
```

Comment:
Nit: It can be helpful to raising an exception when the attributes are not int
```
if isinstance(attr, int):
    optimized_model._c._register_attribute(attr_name, torch._C.IntType.get(), attr)
else:
    raise Exception(f"register_attribute for {attr_name} haven't been implemented")
```

Reply:
Done.